spec/features/recipes/show_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe "ingredients index" do
+  before(:each) do
+    @tomatoe = Ingredient.create!(name: "tomatoe", cost: 2)
+    @lettuce = Ingredient.create!(name: "lettuce", cost: 1)
+    @bread = Ingredient.create!(name: "bread", cost: 5)
+    @bacon = Ingredient.create!(name: "bacon", cost: 10)
+    @ham = Ingredient.create!(name: "ham", cost: 12)
+    
+    @lt = Recipe.create!(name: "LT", complexity: 3, genre: "Veggie Sandwich")
+    @ham_sandwich = Recipe.create!(name: "Ham Sandwich", complexity: 1, genre: "Meat Sandwich")
+    @bacon_sandwich = Recipe.create!(name: "Bacon Sandwich", complexity: 1, genre: "Meat Sandwich")
+
+    @lt_recipe = RecipeIngredient.create!(recipe_id: @lt.id, ingredient_id: @tomatoe.id)
+    @lt_recipe2 = RecipeIngredient.create!(recipe_id: @lt.id, ingredient_id: @lettuce.id)
+    @lt_recipe3 = RecipeIngredient.create!(recipe_id: @lt.id, ingredient_id: @bread.id)
+    @ham_sandwich_recipe = RecipeIngredient.create!(recipe_id: @ham_sandwich.id, ingredient_id: @ham.id)
+    @ham_sandwich_recipe2 = RecipeIngredient.create!(recipe_id: @ham_sandwich.id, ingredient_id: @bread.id)
+    @bacon_sandwich_recipe = RecipeIngredient.create!(recipe_id: @bacon_sandwich.id, ingredient_id: @bacon.id)
+    @bacon_sandwich_recipe2 = RecipeIngredient.create!(recipe_id: @bacon_sandwich.id, ingredient_id: @bread.id)
```

Comment:
Wow, lots of objects created here! That's great, but we really only usually need 2-3 objects per test. If you feel this took you a while to create, you can probably slim down these objects for the next IC.